docs/source/upgrade-guide/index.md

```diff
@@ -186,6 +186,25 @@ becomes:
 import { slateBeforeEach } from '@plone/volto/cypress/support/helpers';
 ```
 
+### StoryBook 8
+
+StoryBook was upgraded from version 6 to 8 in core and in the project generator.
+This section is relevant if you have StoryBook stories in your project or add-on.
+The versions will be upgraded automatically using the `volto-update-deps` script.
+The configuration of your project must also be updated with the new one.
+Replace the `.storybook` folder in your project with this one:
+
+ `https://github.com/plone/volto/tree/main/packages/generator-volto/generators/app/templates/.storybook`
+
+You can find more information about the migration in the [official StoryBook docs](https://storybook.js.org/docs/migration-guide/from-older-version).
```

Comment:
````suggestion

```{seealso}
[Migration guide from Storybook 6.x to 8.0](https://storybook.js.org/docs/migration-guide/from-older-version)
```

````